docs/src/main/java/io/quarkus/docs/generation/YamlMetadataGenerator.java

```diff
@@ -69,7 +73,7 @@ public static void main(String[] args) throws Exception {
                             Document doc = asciidoctor.loadFile(path.toFile(), options);
                             String title = doc.getDoctitle();
                             String id = doc.getId();
-                            Object keywords = doc.getAttribute("keywords");
```

Comment:
the change from keywords to categories is just because of cosmetics, right? does not impact something as we don't yet have real content use it (besides the templates ), right?

Reply:
correct